src/kudu/consensus/leader_election.cc

```diff
@@ -1263,12 +1278,9 @@ void LeaderElection::Run() {
     // Create a single message with comma separated peers
     if (pnum != 0) {
       msg.append(", ");
-      if (pnum % 3 == 0) {
-        msg.append("\n");
-      }
     }
     pnum++;
-    msg.append(state->PeerInfo());
+    msg.append(uuid2hostport(state->peer_uuid, config_));
```

Comment:
What's this part about?

Reply:
I added a function called "uuid2hostport", so log line not only contains uuid, but also host:port

Reply:
Right, for some reason I read this as a rpc message, but this is an log message.